spec/core/SpecSpec.js

```diff
@@ -357,6 +357,23 @@ describe("Spec", function() {
     }]);
   });
 
+  it("should return true when handling an exception", function() {
```

Comment:
I'd have preferred an integration test that tells the overall story, like the various Env integration tests, but I haven't been able to reproduce the overall behavior reliably enough to do that.